applications/luci-app-filetransfer/luasrc/controller/filetransfer.lua

```diff
@@ -8,5 +8,9 @@ Modify: ayongwifi@126.com  www.openwrtdl.com
 module("luci.controller.filetransfer", package.seeall)
 
 function index()
-	entry({"admin", "system", "filetransfer"}, form("updownload"), _("FileTransfer"),89).acl_depends = { "luci-app-filetransfer" }
+
+	local page = entry({"admin", "system", "filetransfer"}, form("filetransfer"), _("FileTransfer"))
+	page.order = 89
```

Comment:
Why split this from entry? Take a look at all of your PRs, you just made this change randomly.